src/connectors/laut.fm.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+Connector.playerSelector = '.fm-player';
+
+Connector.artistSelector = '.player-display__meta .player-display__meta--artist';
+
+Connector.trackSelector = '.player-display__meta .player-display__meta--title';
+
+Connector.isPlaying = () => $('.fm-player__btn').hasClass('playbutton--playing');
```

Comment:
```suggestion
Connector.pauseButtonSelector = '.fm-player__btn.playbutton--playing';
```

Same as the other PR